test/e2e/tests/test_db_cluster.py

```diff
@@ -108,6 +108,53 @@ def aurora_mysql_cluster(k8s_secret):
 
     db_cluster.wait_until_deleted(db_cluster_id)
 
+
+@pytest.fixture
+def aurora_postgres_cluster(k8s_secret):
+    db_cluster_id = random_suffix_name("my-aurora-postgres", 20)
+    secret = k8s_secret(
+        MUP_NS,
+        MUP_SEC_NAME,
+        MUP_SEC_KEY,
+        MUP_SEC_VAL,
+    )
```

Comment:
Thanks @arjunrn ! We should use a different name here to avoid conflict with other created secrets

Reply:
I've fixed the naming. PTAL.